nncf/torch/graph/operator_metatypes.py

```diff
@@ -555,18 +574,20 @@ class PTMatMulMetatype(PTOperatorMetatype):
     }
     hw_config_names = [HWConfigOpName.MATMUL]
     num_expected_input_edges = 2
+    weight_port_ids = [0, 1]
 
 
 @PT_OPERATOR_METATYPES.register()
-class PTBaddBmmMetatype(PTOperatorMetatype):
+class PTAddmmMetatype(PTOperatorMetatype):
     name = "MatMulOp"
-    module_to_function_names = {NamespaceTarget.TORCH: ["baddbmm"]}
+    module_to_function_names = {NamespaceTarget.TORCH: ["addmm"], NamespaceTarget.TORCH: ["baddbmm"]}
     hw_config_names = [HWConfigOpName.MATMUL]
     # 0-th arg to the baddbmm is basically a (b)ias to be (add)ed to the (bmm) operation,
     # presuming that most runtime implementations will fuse the bias addition into the matrix multiplication
     # and therefore won't quantize the bias input, as this would break the hardware-fused pattern.
     ignored_input_ports: List[int] = [0]
     num_expected_input_edges = 2
+    weight_port_ids = [1, 2]
```

Comment:
weight_port_ids  contain 2 but num_expected_input_edges = 2, is it correct?

Reply:
I hope it will be fixed during migration MinMax algorithm to the PyTorch graph with tracing constant.